README.md

```diff
@@ -79,7 +79,7 @@ This action finds the latest Github release semantic version number and generate
 ## How to generate new version
 Let's say current version is `v1.0.0` and you can set `type` equal to one of these values 
 
-- **MAJOR**: It is a major change to your software which may not be compatible with older releases. The new version will be `v2.3.0` 
+- **MAJOR**: It is a major change to your software which may not be compatible with older releases. The new version will be `v3.0.0` 
```

Comment:
I don't think this actually needs to change for the node upgrade, it's just an example of versioning

Reply:
Pretty much all the other actions jumped a version when they went to Node20, so I'm not going to buck the trend.